doc/source_list_library.tex

```diff
@@ -79,8 +81,8 @@ \section*{Appendix: List library}
 // When it encounters a non-list, it applies toString to it.
```

Comment:
Should be `to_string` as defined earlier.

Reply:
noted!